trigger/patch.go

```diff
@@ -389,3 +365,36 @@ func (t *patchTriggers) getGithubContext() (string, error) {
 	}
 	return githubContext, nil
 }
+
+func (t *patchTriggers) patchFamilyOutcome(sub *event.Subscription) (*notification.Notification, error) {
+	if t.data.Status != evergreen.PatchSucceeded && t.data.Status != evergreen.PatchFailed {
+		return nil, nil
+	}
+	if t.event.EventType != event.VersionChildrenCompletion {
+		return nil, nil
+	}
+	return t.generate(sub)
+}
+
+func (t *patchTriggers) patchFamilySuccess(sub *event.Subscription) (*notification.Notification, error) {
+	if t.data.Status != evergreen.PatchSucceeded {
```

Comment:
We can consolidate these 2 cases since they'll both have us return nil, nil

Reply:
done